pipelines/build-testapp.yml

```diff
@@ -94,9 +94,256 @@ stages:
         targetPath: $(artifactDir)
         artifactName: $(artifactName)
 
+  - job: test_app_android
```

Comment:
Please add a condition to disable this job before signing task is ready.

Reply:
disabled the publish task, I think the pipeline can still be run as long as it doesn't produce any artifact